### ISSUE TITLE
instructions in hw9 call for a fork of hw7's repo

### ISSUE THREAD

**Issue by newtypealex** [BE101KG/homework-assignment-instructions#1]
hey @kareemgrant ,  just noticed that the REPO link in the hw9 instructions lead to hw7's repo.  When cloning to my desktop, I already have a hw7 repo, so I not sure what to do.  Did you want us to overwrite our hw7 repo?
https://github.com/BE101KG/homework-assignment-instructions/blob/master/lesson_09.md

However, the REPO link here, https://github.com/BE101KG/homework-assignment-instructions
points to a hw9 repo.

**Comment by kareemgrant** [comment]
Nope, that was my mistake - I've updated the repo link to point to the correct
repo.

Sent using CloudMagic Email
[https://cloudmagic.com/k/d/mailapp?ct=pi&cv=7.6.23&pv=9.2.1&source=email_footer_2]
On Sat, Mar 12, 2016 at 11:28 AM, newtypealex <notifications@github.com> wrote:
hey @kareemgrant [https://github.com/kareemgrant] , just noticed that the REPO link in the hw9 instructions lead to hw7's repo.
When cloning to my desktop, I already have a hw7 repo, so I not sure what to do.
Did you want us to overwrite our hw7 repo?

https://github.com/BE101KG/homework-assignment-instructions/blob/master/lesson_09.md
[https://github.com/BE101KG/homework-assignment-instructions/blob/master/lesson_09.md]

However, the REPO link here, https://github.com/BE101KG/homework-assignment-instructions
[https://github.com/BE101KG/homework-assignment-instructions]
points to a hw9 repo.

—
Reply to this email directly or view it on GitHub
[https://github.com/BE101KG/homework-assignment-instructions/issues/1] .[https://github.com/notifications/beacon/ABkDE0IVfkCRpyzN81PlWb-ymKj9aB-jks5psumsgaJpZM4HvVcQ.gif]
